### PR TITLE
Feat separate processes by group in selective newsletter

### PR DESCRIPTION
**Pull request by aitorlb**
#### :tophat: What? Why?
Separate `participatory_processes` by `process_group` in `select_recipients_to_deliver` in the admin's newsletter page.

#### :pushpin: Related Issues
- Related to #50

#### :clipboard: Subtasks
- [x] Add `CHANGELOG` entry

#### Newsletter before
newsletter_before

#### Newsletter after
newsletter_after

### CODE REVIEW

**Comment by tramuntanal** [comment]
@aitorlb changelog entry please

File: Gemfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 source "https://rubygems.org"
 
 ruby RUBY_VERSION
-DECIDIM_VERSION = { git: 'https://github.com/gencat/decidim', tag: 'v0.18-dev-gencat-features' }.freeze
+DECIDIM_VERSION = { git: 'https://github.com/gencat/decidim', tag: 'v0.18-dev-gencat-features-fix' }.freeze
```
Comment (tramuntanal) [inline Gemfile]: shouldn't we merge your fix into "v0.18-dev-gencat-features"? instead of creating a new branch?

Comment (tramuntanal) [reply Gemfile]: @isaacmg410 what do you think?

File: config/locales/ca_admin.yml [hunk 2]
```diff
@@ -3,3 +3,12 @@ ca:
     attributes:
       attachment_collection:
         weight: Pes
+  decidim:
```
Comment (tramuntanal) [inline config/locales/ca_admin.yml]: do we need to change this translations only to split the multiselect in two?

File: app/decorators/decidim/admin/selective_newsletter_form_decorator.rb [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+Decidim::Admin::SelectiveNewsletterForm.class_eval do
+  def map_model(_newsletter)
+    self.participatory_space_types = Decidim.participatory_space_manifests.map do |manifest|
+      Decidim::Admin::SelectiveNewsletterParticipatorySpaceTypeForm.from_model(manifest: manifest)
+    end.unshift(additional_participatory_space_manifest)
+  end
```
Comment (tramuntanal) [inline app/decorators/decidim/admin/selective_newsletter_form_decorator.rb]:
```suggestion
self.participatory_space_types.unshift(additional_participatory_space_manifest)
```

File: config/locales/ca_admin.yml (hunk 2 above)
Comment (aitorlb) [reply config/locales/ca_admin.yml]: Oops. Should I make another PR?

Comment (isaacmg410) [reply config/locales/ca_admin.yml]: Yes do it please.

File: app/decorators/decidim/admin/selective_newsletter_form_decorator.rb (hunk 3 above)
Comment (aitorlb) [reply app/decorators/decidim/admin/selective_newsletter_form_decorator.rb]: Like this? Rubocop complains about:  Redundant `self` detected.
```ruby
  def map_model(_newsletter)
    self.participatory_space_types = Decidim.participatory_space_manifests.map do |manifest|
      Decidim::Admin::SelectiveNewsletterParticipatorySpaceTypeForm.from_model(manifest: manifest)
    end
    participatory_space_types.unshift(additional_participatory_space_manifest)
  end
```

File: config/locales/ca_admin.yml (hunk 2 above)
Comment (aitorlb) [reply config/locales/ca_admin.yml]: Done: #90

File: app/decorators/decidim/admin/selective_newsletter_form_decorator.rb (hunk 3 above)
Comment (tramuntanal) [reply app/decorators/decidim/admin/selective_newsletter_form_decorator.rb]: yes, I think it's much more readable and easy to understand this way

Comment (aitorlb) [reply app/decorators/decidim/admin/selective_newsletter_form_decorator.rb]: Thanks. Done.

File: Gemfile (hunk 1 above)
Comment (isaacmg410) [reply Gemfile]: This is not a new branch. Is a new tag. As we discussed yesterday we will work with the branch system.
@aitorlb can you change the tag to the branch: `newsletter_fix_and_admin_department_refactor` please?

**Pull request merged**